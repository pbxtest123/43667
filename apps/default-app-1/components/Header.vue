<template>
    <h1>Klient template - header</h1>
    <Button />
</template>

<script setup>
import {Button} from "#components"
console.log('Uruchomienie skryptu z klienta')
</script>