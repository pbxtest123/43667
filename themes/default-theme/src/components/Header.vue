<template>
  <h1>Default template - header</h1>
  <Button />
</template>

<script setup>
import {Button} from "#components"
console.log('Uruchomienie skryptu z theme')

// const r = useRouter();
// console.log({r})
</script>