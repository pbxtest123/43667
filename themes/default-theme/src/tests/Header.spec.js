import {mount} from "@vue/test-utils";
import Header from "../components/Header.vue";
// import { expect, test } from "vitest";
const wrapper = mount(Header);

it("testing GuessAge component props", async () => {
    // expect(wrapper.text()).toMatchSnapshot()
});

// import { describe, test } from 'vitest'
// import { setup, $fetch } from '@nuxt/test-utils'
//
// describe('My test', async () => {
//     await setup({
// nuxtConfig: {}
//         // test context options
//     })
//
//     test('my test', () => {
//         const wrapper = mount(Header);
//         console.log(wrapper)
//         // ...
//         expect(2+2).toBe(4)
//     })
// })
