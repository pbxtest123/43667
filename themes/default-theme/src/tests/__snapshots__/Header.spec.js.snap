// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`testing GuessAge component props 1`] = `"Default template - header"`;
